tests/test_buffer.py

```diff
@@ -436,6 +437,22 @@ def test_SimpleArray_from_ndarray(self):
         sarr_from_cpp = modmesh.SimpleArrayFloat64(shape=(2, 3, 4))
         self.assertFalse(sarr_from_cpp.is_from_python)
 
+        shape = (2, 3, 5, 7)
+        np_sarr = np.array(shape=shape, dtype='float64')
```

Comment:
Use `np.array` instead of `np.ndarray`.  The latter is not a numpy coding convention. We do not usually directly call the `ndarray` constructor. Instead, we use the helpers `array`, `empty`, `ones`, `zeros`, `full`, etc.